package.json

```diff
@@ -29,11 +31,15 @@
     "@types/node": "^14.14.22",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "did-jwt": "~5.1.2",
+    "did-resolver": "^3.0.1",
+    "dids": "^2.1.0",
     "gh-pages": "^3.1.0",
     "key-did-provider-ed25519": "^1.0.0",
+    "key-did-resolver": "^1.2.0",
```

Comment:
This should be a regular dep?

Reply:
yup.

Reply:
This is a static application, so as initiated by @PaulLeCam all of the dependencies are regular ones.